src/main/java/seedu/address/model/lesson/Day.java

```diff
@@ -1,8 +1,28 @@
 package seedu.address.model.lesson;
 
+import seedu.address.model.lesson.exceptions.InvalidDayException;
+
 /**
  * Represents the days in a week.
  */
 public enum Day {
-    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY
+    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY;
+
+    public static final String MESSAGE_CONSTRAINTS = "Day must be typed out fully in capital letters.\n"
```

Comment:
Not too sure about this, but "Days" sound slightly better? Or perhaps can consider using "The day...", but it might not sit too well with the other constraint messages.
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Days must be typed out fully in capital letters.\n"
```

Reply:
Perhaps to keep it consistent with other constraints message we can use 
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Days should only contain capital letters.\n"
```